### PR TITLE
integration.admin-client: Add exact search for all dedicated user attributes

### LINKED ISSUE #13360: integration.admin-client: Add exact search for all dedicated user attributes
### Description

Right now the admin-client supports exact search for username but not for lastname,forename and email while those support it too.

I submitted a PR to fix the behaviour #13361

### PR DESCRIPTION
**Pull request by bs-matil**
a trivial commit to add exact search for all user attributes in the UsersResource for the admin-client see #13360

### CODE REVIEW

File: integration/admin-client/src/main/java/org/keycloak/admin/client/resource/UsersResource.java [hunk 1]
```diff
@@ -127,6 +127,22 @@ List<UserRepresentation> searchByAttributes(@QueryParam("first") Integer firstRe
...
+    @Produces(MediaType.APPLICATION_JSON)
+    List<UserRepresentation> searchByUsername(@QueryParam("username") String username, @QueryParam("exact") Boolean exact);
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    List<UserRepresentation> searchByEmail(@QueryParam("email") String email, @QueryParam("exact") Boolean exact);
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    List<UserRepresentation> searchByFirstName(@QueryParam("firstName") String email, @QueryParam("exact") Boolean exact);
```
Comment (danielFesenmeyer) [inline integration/admin-client/src/main/java/org/keycloak/admin/client/resource/UsersResource.java:140]: first param should better be named "firstName". Also see next method ("lastName"),

**Comment by bs-matil** [comment]
@mposolda any chance to get this upstream?

**Comment by mposolda** [comment]
@bs-matil Yes, this should work. Do you have a chance to add some automated test for the new methods? Probably some new test in the class `UsersTest` or `UserTest` should work?

**Comment by bs-matil** [comment]
I will have a look thanks

**Review by mposolda — APPROVED** [review]
@bs-matil Thanks!

**Pull request merged**